index.html

```diff
@@ -70,7 +71,7 @@ <h2>Looking for your next Surf Trip?</h2>
       <button class="email-btn">Email me!</button>
     </div>
     <footer>
```

Comment:
. Link 'rel' attribute should include 'noopener',  just like the example below:
</footer>